apps/explorer/src/hooks/useGetTransactionBlocks.ts

```diff
@@ -32,7 +32,10 @@ export function useGetTransactionBlocksForAddress(
         {
             getNextPageParam: (lastPage) =>
                 lastPage?.hasNextPage ? lastPage.nextCursor : false,
-            enabled: !!address,
+            staleTime: Infinity,
+            cacheTime: 24 * 60 * 60 * 1000,
```

Comment:
I know this is just copied over but I'd remove this and let the default flow through